frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -443,30 +257,28 @@ const composeAndPushNotificationMessages = (
                     <label>Assign a role to the user</label>
 
                     <Field
-                        name="role_id"
+                        name="roleId"
                         aria-label="Role Select"
                         v-slot="{ field, handleChange }"
-                        v-model="formData.role_id"
+                        v-model="formData.roleId"
                     >
                         <Dropdown
                             :options="applicationRoleOptions"
                             optionLabel="role_name"
                             optionValue="role_id"
                             :modelValue="field.value"
                             placeholder="Choose an option"
-                            class="w-100 custom-input"
+                            class="w-100 custom-height"
                             style="width: 100% !important"
                             v-bind="field.value"
                             @update:modelValue="handleChange"
-                            @change="removeForestClientSection()"
+                            @change="resetVerifiedForestClients"
```

Comment:
we need this line to remove the verified forest client number from the formData when role changed